test/javaserver/Requests/HttpRequestParserTest.java

```diff
@@ -2,17 +2,30 @@
 
 import org.junit.Test;
 
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
 
 public class HttpRequestParserTest {
 
     private String request = "GET / HTTP/1.1 \r\n";
+
     @Test
-    public void testGetsTheCorrectVerb() {
+    public void testGetsTheCorrectVerbGet() {
         String verb = "GET";
         HttpRequestParser requestParser = new HttpRequestParser(request);
 
-        assertEquals(verb, requestParser.getVerb());
+        assertEquals(verb, requestParser.httpMethod());
+    }
+
+    @Test
+    public void testGetsTheCorrectVerbPost() {
+        String verb = "POST";
```

Comment:
It's probably a bit overkill to have a variable for a value that is only used in one place, three or four lines later. I would probably inline it.

I would also consider extracting constants for the various HTTP methods. e.g. `HTTPMethods.GET` and `HTTPMethods.POST` or something like that.